src/annotator/config.js

```diff
@@ -18,7 +18,7 @@ function config(window_) {
 
   // Parse config from `<script class="js-hypothesis-config">` tags
   try {
-    Object.assign(options, settings(window_.document));
+    Object.assign(options, settings.jsonConfigsFrom(window_.document));
```

Comment:
I think this is _much_ clearer, `jsonConfigsFrom(document)` tells me that this is reading the JSON objs from the document, without me having to go look in `settings.js`